templates/account.html

```diff
@@ -44,15 +43,22 @@ <h3>My snaps</h3>
                   {% else %}
                     {% for snap in user_snaps|sort %}
                         <tr>
-                            <td>
+                            <td class="u-vertically-center">
+                                {% if user_snaps[snap].icon_url %}
+                                    <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" />&nbsp;
```

Comment:
`width="32" height="32"` could it be in a css file instead

Reply:
There isn't any css for this page and it's getting redesigned so I don't think it's worth creating a new css file just for this - vanilla all the way.